File: PRAK402-2210817120011-MAULIDASARI.c
#include<stdio.h>

int main() {
int i, j, batas;

printf ("Masukan batas maksimal: ");
scanf("%d", &batas);

for(i=1; i<=batas; i+=2) {
   printf("%d", i);
   printf(" ", i);
}
   printf("\n");
for(j=batas; j>=1; j--) {
    if(j%2==0){
       printf("%d", j);
       printf(" ", i);
    }
  }
}

File: PRAK402-2210817120011-MAULIDASARI.py
batas=int(input("Masukan batas maksimal : "))
for i in range(1, batas+1):
    if i%2!=0 :
        print(i, end=' ')
    else:
        pass
    print(' ')
    for i in reversed(range(2, batas+1)):
        if i%2==0 :
            print(i, end=' ')
        else:
            pass